add kvm group in system gid range

as expected by the postinst of Debian udev package.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Abdur Rehman <abdur.rehman@toradex.com>
---
 group.master | 1 +
 1 file changed, 1 insertion(+)

diff --git a/group.master b/group.master
index cea9d60..5b62284 100644
--- a/group.master
+++ b/group.master
@@ -34,6 +34,7 @@ utmp:*:43:
 video:*:44:
 sasl:*:45:
 plugdev:*:46:
+kvm:*:102:
 staff:*:50:
 games:*:60:
 shutdown:*:70:
